watertap/unit_models/tests/test_pressure_exchanger.py

```diff
@@ -137,17 +167,28 @@ def test_build():
                 assert isinstance(sb_exp, Expression)
 
     # test statistics
-    assert number_variables(m.fs.unit) == 39
-    assert number_total_constraints(m.fs.unit) == 31
+    assert number_variables(m.fs.unit) == 39 + extra_variables
+    assert number_total_constraints(m.fs.unit) == 31 + extra_constraint
     assert number_unused_variables(m.fs.unit) == 0
 
+
+@pytest.mark.unit
+def test_build_with_mass_transfer():
+    # 2 variables for mass tranfer fractions (solute,solvent)
+    # 4 extra vars for mass_transfer_term on LP and HP side, and solute, solvent
+    # 4 extra constraints for mass transsfer from HP side (soute, solvent)
+    test_build(has_mass_transfer=True,\
+    extra_variables=6,\
+    extra_constraint=4)
+
 class TestPressureExchanger():
     @pytest.fixture(scope="class")
     def unit_frame(self):
         m = ConcreteModel()
         m.fs = FlowsheetBlock(default={'dynamic': False})
         m.fs.properties = props.SeawaterParameterBlock()
```

Comment:
This might cause some headaches and could be reserved for a subsequent PR, but wondering if we'd want another test that uses a property package with multiple ions.

Reply:
For example, you could try using the `seawater_ion_generic` property model, which is based on the generic property framework in IDAES; this is likely overkill and issues may arise when trying this, but it includes 5 ions representative of seawater.

Reply:
Will give it a go and probably should address any issues in this PR, as we will likely want to use this model with more complex property packages... Let me see what issues crop up.

Reply:
Revisiting `seawater_ion_generic`, I recalled that a molar flow basis is used, which might cause the first upset. You can give it a try (checkout `test_nanofiltration_ZO` to see how `seawater_ion_generic` gets imported and loaded into a `GenericParameterBlock`), but maybe there's an easier way: you can try @andrewlee94 's new `zero_order_properties` located in the `core` folder. This way, you can just declare a solute_list when you build the property model. However, things might still break if certain properties don't match between the unit model and property model.

Reply:
Latest commit fixed the issue.